sunpy/io/special/asdf/resources/schemas/geocentricearthequatorial-1.0.0.yaml

```diff
@@ -1,14 +1,15 @@
 %YAML 1.1
 ---
 $schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
-id: "asdf://sunpy.org/schemas/geocentricearthequatorial-1.0.0"
+id: "asdf://sunpy.org/sunpy/schemas/geocentricearthequatorial-1.0.0"
 
 title: |
   Represents a Geocentric Earth Equatorial (GEI) coordinate object from SunPy
 
 examples:
   -
     - A GeocentricEarthEquatorial frame without data
+    - asdf-standard-1.5.0
```

Comment:
Is this something which is affecting how the tests verify this?

Reply:
I went looking for documentation on this and found none :facepalm:

This tells the `pytest-asdf` plugin to run this example using ASDF standard 1.5.0. Here's a link to the code that handles this (although it's not that helpful):
https://github.com/asdf-format/asdf/blob/48c5c57bb8a471a3bad7f2d5791a9576422abeb5/pytest_asdf/plugin.py#L184

This (and the other similar additions) is needed as the example uses a tag that is not in ASDF standad 1.6.0. In this example, the following time tag is used:
```yaml
equinox: !time/time-1.1.0
```
which exists in 1.5.0 but not in 1.6.0 (which uses `time-1.2.0` instead). Without this change, when 1.6.0 becomes the default, this example will start failing because `time-1.1.0` is not a valid 1.6.0 tag.

Let me know if more details are helpful or if you have any questions.

Reply:
That is what I assumed, thanks for confirming.